code/__DEFINES/flags.dm

```diff
@@ -63,6 +63,7 @@
 #define TAIL_WAGGING    	1024
 #define NO_EYES				2048
 #define HAS_FUR				4096
+#define NO_ABSORB			8192
 
```

Comment:
Not sure about NO_ABSORB here, I think it should be under species flags, doesn't really fit in with the body ones